src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -3006,6 +3007,21 @@ GUI_TEST_CLASS_DEFINITION(test_7644) {
     GTUtilsTaskTreeView::waitTaskFinished(os);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7645) {
+    //1. Open file "/_common_data/primer3/7645.seq"
+    //2. Run Primer 3 with default parameters
+    //Expected state: there is only two files in project view - sequence and annotations
+    GTFileDialog::openFile(os, testDir + "/_common_data/primer3", "7645.seq");
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+
+    GTUtilsDialog::add(os, new PopupChooser(os, {"ADV_MENU_ANALYSE", "primer3_action"}));
+    GTUtilsDialog::add(os, new Primer3DialogFiller(os));
+    GTWidget::click(os, GTUtilsSequenceView::getPanOrDetView(os), Qt::RightButton);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    CHECK_SET_ERR(GTUtilsProjectTreeView::countFilesInProjectView(os) == 2, "two opened files expected");
```

Comment:
Can you check the items by names here too, so it is not random items, but the expected ones?

Reply:
I used this solution because we cant be sure about generated names in default directory for annotations. But we are sure about quantity of files in project after  performing Primer3 execution.

Reply:
Every test is run in isolation and tests are reproducible. 
It means that the produced file names in the isolated environment are the same.
Why don't you know the generated file names? If there is a big problem with it check part of the names &types.